### PR TITLE
Feat/cli gui

### LINKED ISSUE #181: Be able to query the basic agent state via the GUI
**Is your feature request related to a problem? Please describe.**
Currently, the GUI does not allow for querying the agent state.

**Describe the solution you'd like**
Display the agent state (`AgentState`) via the GUI and let the user terminate (`CTRL+C`) an agent again from GUI.

**Describe alternatives you've considered**
-

**Additional context**
-

### PR DESCRIPTION
**Pull request by dishmop**
## Proposed changes

Can start and stop OEF nodes and agents from gui.
Can launch gui from cli

## Fixes

#181

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [x] I have read the [CONTRIBUTING](../master/CONTRIBUTING.rst) doc
- [x] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that the documentation about the `aea cli` tool works
- [ ] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: aea/cli_gui/__init__.py [hunk 1]
```diff
@@ -0,0 +1,425 @@
+# -*- coding: utf-8 -*-
```
Comment (marcofavorito) [inline aea/cli_gui/__init__.py]: Looks good. But if we could create a `aea/cli_gui/__main__.py` file with some relevant parts of `aea/cli_gui/__init__.py` would be the best. With that change, we could run `python -m aea.cli_gui`.

Indeed, if I execute that command now:
```
$ python -m aea.cli_gui
/home/marcofavorito/.virtualenvs/agents-aea-jzIedExH/bin/python: No module named aea.cli_gui.__main__; 'aea.cli_gui' is a package and cannot be directly executed
```

Comment (dishmop) [reply aea/cli_gui/__init__.py]: Hi Marco, 

If I have the run function (or nay function) in __init__.py can I call them from __main__.py?  how do I do that? Do I have to import something into __main__.py

Comment (marcofavorito) [reply aea/cli_gui/__init__.py]: In particular, we could move [these lines](https://github.com/fetchai/agents-aea/pull/223/files/45c9252b638a1ba5e4b8f344d44ab399d1846f90#diff-bb1b8b3aa7790d5967a26dded4a97fb4R387-R425) into `aea/cli_gui/__main__.py`

**Comment by dishmop** [comment]
Probably don't review this code at the moment as it currently doesn't t run properly - giving errors when I try and create a new agent.

**Pull request merged**